programs/token-metadata/program/tests/update.rs

```diff
@@ -177,12 +175,96 @@ mod update {
 
         // checks the created metadata values
         let metadata = da.get_metadata(context).await;
+        let original_update_authority = Keypair::from_bytes(&context.payer.to_bytes()).unwrap();
 
-        assert_eq!(metadata.update_authority, delegate.pubkey());
+        assert_eq!(
+            metadata.update_authority,
+            original_update_authority.pubkey()
+        );
```

Comment:
This is fine since its a test and I believe you are just being explicit, but FYI above there's already an `update_authority` Keypair that doesn't change during the test.

Reply:
We can't use the `update_authority` at this point, the value was moved in line `128` when the delegate is created.